public/api-docs/swagger.json

```diff
@@ -107,83 +107,39 @@
         }
       }
     },
-    "/auth/resetpassword":{
```

Comment:
Please check if you did not delete this route while rebasing

Reply:
I think I did, Let me bring it back